test/app/components/without_validation_component.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class WithoutValidationComponent < ActionView::Component::Base
```

Comment:
Its feels a little strange to have a `WithoutValidationComponent` that has a `validates` method. This isn't how you'd do it in your application and I worry about test code as misleading documentation. 

Instead how about we override the `validate!` method in this class and have it raise

```ruby
def validate!
  raise "validate! was called"
end
```

Reply:
agree